### DISCUSSION TITLE
galaxy brain test

### DISCUSSION THREAD

**Discussion by 1wos** [Schweinepriester/achievement-testarea#36] [Q&A] [answered]
I want to experiment with this!

**Comment by seo-asif** [comment]
Me too... Please accept my answer

**Reply by 1wos** [reply]
Can you post my repository discord questions for the galaxy brain test?  Can you post a question on the Q&A? Then I'll help you through adoption! 

https://github.com/sommizzu/Big-Data-Convergence/discussions

**Comment by hoseinpur** [comment, marked as answer]
ok, let's try in this post @sommizzu

**Reply by 1wos** [reply]
I chose your answer!

**Reply by hoseinpur** [reply]
should my answer be marked as an answer only once?? because I still didn't get the galaxy brain! could you please mark this as an answer too?

**Reply by 1wos** [reply]
@hoseinpur Only the adoption through answer to the Q&A post can receive the Galaxy Brine. I'll post a new Q&A post, so please comment there.

**Reply by 1wos** [reply]
@hoseinpur https://github.com/Schweinepriester/achievement-testarea/discussions/37
If you reply here, I'll adopt it. Then two questions are answered. 
You'll get a Galaxy Brain!

**Reply by hoseinpur** [reply]
ok got it, so this is my reply here. and I also replied in the new post!
